common/src/androidTest/java/com/microsoft/identity/common/internal/platform/DevicePoPManagerTests.java

```diff
@@ -289,4 +302,60 @@ public void testMintSignedAccessTokenContainsRequisiteClaims()
         Assert.assertEquals(nonce, jwtClaimsSet.getClaim("nonce"));
         Assert.assertNotNull(jwtClaimsSet.getClaim("cnf"));
     }
+
+    @Test
+    public void testAsymmetricKeyNullWhenUninitialized() throws ClientException {
```

Comment:
nit: rename to `testAsymmetricKeyCreationDateNullWhenUninitialized`

Reply:
See cf890e25e09cf5beb74c4c0a10f9a206ac7af973 thanks!